Ravel-Creators/Assets/RavelCreator/Scripts/Editor/CreatorTools/Config/RavelCreatorSettingsEditor.cs

```diff
@@ -0,0 +1,58 @@
+#if UNITY_EDITOR
+using System;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+public partial class RavelCreatorSettings
+{
+	[CustomEditor(typeof(RavelCreatorSettings))]
+	public class RavelCreatorSettingsEditor : Editor
+	{
+		private const float BTN_WIDTH = 100;
+		
+		private RavelCreatorSettings _instance;
+
+		private void OnEnable() {
+			_instance = (RavelCreatorSettings)target;
+		}
+
+		public override void OnInspectorGUI() {
+			DrawDefaultInspector();
+
+			GUIDrawFolderAndSetBtn(_instance._filePath, _instance.SetFilePath,
```

Comment:
I added this in the tooltip, but the settings file is not really for direct usage, as all these values are set through the editor windows. If you do set a wrong folder it will throw a warning though.